eigenda/config.go

```diff
@@ -88,27 +85,65 @@ func CLIFlags(envPrefix string) []cli.Flag {
 	}
 	return []cli.Flag{
 		&cli.StringFlag{
-			Name:    RPCFlagName,
-			Usage:   "RPC endpoint of the EigenDA disperser.",
-			EnvVars: prefixEnvVars("TARGET_RPC"),
+			Name:     RPCFlagName,
+			Usage:    "RPC endpoint of the EigenDA disperser.",
+			EnvVars:  prefixEnvVars("TARGET_RPC"),
+			Required: true,
+		},
+		&cli.DurationFlag{
+			Name:     StatusQueryTimeoutFlagName,
+			Usage:    "Timeout for aborting an EigenDA blob dispersal if the disperser does not report that the blob has been confirmed dispersed.",
+			Value:    30 * time.Minute,
+			EnvVars:  prefixEnvVars("TARGET_STATUS_QUERY_TIMEOUT"),
+			Required: false,
 		},
 		&cli.DurationFlag{
-			Name:    StatusQueryTimeoutFlagName,
-			Usage:   "Timeout for aborting an EigenDA blob dispersal if the disperser does not report that the blob has been confirmed dispersed.",
-			Value:   25 * time.Minute,
-			EnvVars: prefixEnvVars("TARGET_STATUS_QUERY_TIMEOUT"),
+			Name:     StatusQueryRetryIntervalFlagName,
+			Usage:    "Wait time between retries of EigenDA blob status queries (made while waiting for a blob to be confirmed by).",
+			Value:    5 * time.Second,
+			EnvVars:  prefixEnvVars("TARGET_STATUS_QUERY_INTERVAL"),
+			Required: false,
+		},
+		&cli.BoolFlag{
+			Name:     DisableTlsFlagName,
+			Usage:    "Disable TLS when connecting to the EigenDA disperser.",
+			Value:    false,
+			EnvVars:  prefixEnvVars("TARGET_GRPC_DISABLE_TLS"),
+			Required: false,
 		},
 		&cli.DurationFlag{
-			Name:    StatusQueryRetryIntervalFlagName,
-			Usage:   "Wait time between retries of EigenDA blob status queries (made while waiting for a blob to be confirmed by).",
-			Value:   5 * time.Second,
-			EnvVars: prefixEnvVars("TARGET_STATUS_QUERY_INTERVAL"),
+			Name:     ResponseTimeoutFlagName,
+			Usage:    "The total amount of time that the client will waiting for a response from the EigenDA disperser.",
+			Value:    10 * time.Second,
+			EnvVars:  prefixEnvVars("RESPONSE_TIMEOUT"),
+			Required: false,
+		},
+		&cli.UintSliceFlag{
+			Name:     CustomQuorumIDsFlagName,
+			Usage:    "The quorum IDs to write blobs to using this client. Should not include default quorums 0 or 1.",
+			Value:    cli.NewUintSlice(),
+			EnvVars:  prefixEnvVars("CUSTOM_QUORUM_IDS"),
+			Required: false,
+		},
+		&cli.StringFlag{
+			Name:     SignerPrivateKeyHexFlagName,
+			Usage:    "Signer private key in hex encoded format. This key should not be associated with an Ethereum address holding any funds.",
```

Comment:
Let's put a major disclaimer about expected key mgmt practices in readme for this variable

Reply:
also might be worth disclosing to our customers how they can obtain authorization with EigenDA for reduced rate limiting requirements